package nextstep.di.exception;

public class ReflectionUtilException extends RuntimeException {
    public ReflectionUtilException(String message, Throwable cause) {
        super(message, cause);
    }
}
